python/lance/pytorch/data.py

```diff
@@ -30,23 +31,29 @@
 
 import lance
 from lance import dataset
+from lance.types.image import is_image
 
 __all__ = ["LanceDataset"]
 
 
-def to_numpy(arr: pa.Array):
-    """Convert pyarrow array to numpy array"""
+def to_tensor(arr: pa.Array) -> Union[torch.Tensor, PIL.Image.Image]:
```

Comment:
Just double checking that since we're passing in batches here these are guaranteed not to be ChunkedArray right?

Reply:
Each RecordBatch returned from scanner is a continue array.